config/image/oss/kustomization.yaml

```diff
@@ -7,4 +7,4 @@ images:
   newTag: '2.8'
 - name: kic-placeholder
   newName: kong/kubernetes-ingress-controller
-  newTag: '2.5.0'
+  newTag: '2.5.1'
```

Comment:
In https://github.com/Kong/kubernetes-ingress-controller/pull/4339 we've introduced a change to require manifests to be of `{major}.{minor}` format. I don't think we need to do this here but just leaving this for posterity

Reply:
It wasn't in main when we cut this version, so it falls into the big bin of things that don't work the same in backports as they do in modern releases.

Long term we probably need to figure out a more comprehensive means of managing LTS so it has more modern CI/CD scaffolding, since this one has demonstrated that trying to leave everything as it was originally results in a lot of problems.